docs/api/scripts_refactor_parsers.md

```
@@ -0,0 +1,128 @@
+**Parameters:**
+results (Dict[str, Dict[str, Any]]): Docstring analysis results.
+output_dir (Path): Directory to write markdown files to.
+
+#### `_generate_module_markdown`
+Generate markdown documentation for a single module.
+**Parameters:**
+file_path (str): Path to the module.
+docstrings (Dict[str, Any]): Docstring information for the module.
+output_path (Path): Path to write markdown file to.
+
+#### `__init__`
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)